question_answering/question_answering.py

```diff
@@ -278,22 +303,25 @@ def answer_questions(
     # Prepare the successes dataframe and errors dictionary to be returned:
     successes = []
     errors = {}
+
     # Split the files into batches:
     file_batches = [
         text_files[i : i + batch_size]
         if i + batch_size < len(text_files)
         else text_files[i:]
         for i in range(0, len(text_files), batch_size)
     ]
-    questions_config = _list_me(
-        questions_config, "questions_config", number_of_question_groups
+    questions_config = _to_group_list(
+        argument_value=questions_config,
+        argument_name="questions_config",
+        length=number_of_question_groups,
     )
-    # Create a list of question handlers according to given configs
 
+    # Create a list of question handlers according to given configs
     handlers = []
     for cfg in questions_config:
-        question_type = cfg.get("type") or "default"
-        handlers.append(QUESTION_MAPPING.get(question_type)(cfg))
+        question_type = cfg.get("type", "default")
```

Comment:
```suggestion
        question_type = cfg.pop("type", "default")
```
This will remove the key from the config dictionary